### PR TITLE
express-static - James Yang

### PR DESCRIPTION
**Pull request by jamesdae**
static css js
static get

### CODE REVIEW

File: express-static/index.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+const express = require('express');
+const path = require('path');
+const app = express();
+app.use(express.static('public'));
+
+const joined = path.join('__dirname', 'public');
```
Comment (Zhiwutian) [inline express-static/index.js:6]: this should be called with __dirname, the variable

File: express-static/index.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+const express = require('express');
+const path = require('path');
+const app = express();
+app.use(express.static('public'));
```
Comment (Zhiwutian) [inline express-static/index.js:4]: use should be called with the `joined` variable below once you have set the path

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
a couple updates

**Comment by jamesdae** [comment]
https://user-images.githubusercontent.com/110082070/208489842-9003584b-88a1-4805-bce7-9b02bdb6c3c3.mov

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**